javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,96 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const doubledOddNumbers = numbers.filter((x) => x % 2 !== 0).map((x) => x * 2);
+console.log(doubledOddNumbers);
+
+//movies with title name consists of 1 word (short title names)
+const shortTitledMovies = movies.filter(
+  (movie) => movie.title.split(" ").length < 2
+);
+console.log(shortTitledMovies);
+
+//movies with title name consists of more than 8 words (long title names)
+const longTitledMovies = movies.filter(
+  (movie) => movie.title.split(" ").length > 8
+);
+console.log(longTitledMovies);
+
+//movies between 1980-1989
+let n = 0;
+movies.forEach((movie) => {
+  if (movie.year < 1990 && movie.year > 1979) {
```

Comment:
Good stuff! Remember that we also have "greater/less than or equal to" operators, so the clause could be `movie.year >= 1980 && movie.year <= 1989`. You don't need to change this, just a heads up